src/cljc/quil/snippets/lights_camera/camera.cljc

```diff
@@ -75,23 +75,27 @@
 
 (defsnippet ortho-perspective
   ["ortho" "perspective"]
-  {:renderer :p3d
-   :setup (q/frame-rate 1)}
+  {:renderer :p3d}
 
   (q/background 240)
-  (comment "flip between ortho and perspective camera every frame")
-  (comment "enable ortho camera")
+
+  (comment "set camera to look from [400, 100, 200] at point [100, 0, 0]")
+  (q/camera 400 100 200 100 0 0 0 0 -1)
+
   (comment "in ortho all figures will look the same regardless distance")
+  (q/ortho)
+  (comment "draw 3 red boxes with x coordinates 0, 100 and 200")
+  (q/fill 200 0 0)
+  (doseq [x [0 100 200]]
+   (q/with-translation [x 50 0]
+     (q/box 50)))
+
   (comment "in perspective (default) all figures will look smaller the farther they are")
-  (if (even? (q/frame-count))
-    (q/ortho)
-    (q/perspective))
-  (comment "set camera to look from [300, 0, 300] at point [100, 0, 0]")
-  (q/camera 300 0 300 100 0 0 0 0 -1)
-  (q/fill 0 127 127)
-  (comment "draw 3 boxes with x coordinates 0, 100 and 200")
+  (q/perspective)
```

Comment:
I'm confused a little bit. How can you have both ortho and perspective cameras in the same image?

Reply:
My guess is also that it is not intended to be used that way. In p5 docs both calls to [perspective](https://p5js.org/reference/#/p5/perspective) and [ortho](https://p5js.org/reference/#/p5/ortho) are made in `setup`. But since it's not a must we can display both side by side. I was hoping fill is enough to distinguish but if you think this is anyway confusing perhaps it makes sense to have 3 snippets as you suggested earlier?
* 1 with static image showing ortho (for automated tests)
* 1 with static image showing perspective (for automated tests)
* 1 with dynamic image showing switch between perspective and ortho (for use on quil.info)

Reply:
But fill actually removes all  (q/ortho) boxes that were drawn before that, right? Yes, I think having two separate snippets will help to avoid the issue entirely.

Reply:
No, just checked to be sure, both clj and cljs look like this:
<img width="568" alt="Screenshot 2019-05-18 at 10 33 33" src="https://user-images.githubusercontent.com/1301852/57966930-c888c400-7958-11e9-91c4-604eb8b6e9ee.png">

Reply:
For determining whether to run automated image comparison on a snippet, what do you think about adding an opt in the snippet? Something like this:
```clj
(defsnippet ortho-perspective
  ["ortho" "perspective"]
  {:renderer :p3d
   :automated-comparison? false}
```
with default `true`. Then we could get rid of the function `skip-automated-compare?`.

Reply:
> No, just checked to be sure, both clj and cljs look like this:
Thank you for providing the screenshot. I was wrong.

> For determining whether to run automated image comparison on a snippet, what do you think about adding an opt in the snippet?
That sounds good. Better than having a whitelist in a separate file. One note though that it would be better to have the field "false" by default. Maybe `:skip-automated-compare? true`. The reason is that usually options are off by default and you turn then on by explicitly setting it. It's just slightly easier to parse mentally.